Source/Frontend/UI/Components/Glitch Harvester/RTC_StockpileManager_Form.designer.cs

```diff
@@ -33,7 +33,7 @@ private void InitializeComponent()
             System.Windows.Forms.DataGridViewCellStyle dataGridViewCellStyle4 = new System.Windows.Forms.DataGridViewCellStyle();
             System.Windows.Forms.DataGridViewCellStyle dataGridViewCellStyle2 = new System.Windows.Forms.DataGridViewCellStyle();
             System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(RTC_StockpileManager_Form));
-            this.dgvStockpile = new RTCV.UI.DataGridViewDraggable();
+            this.dgvStockpile = new RTCV.UI.Components.DataGridViewDraggable();
```

Comment:
I manually fixed the namespacing for this designer file. Should I leave the namespace as it was to avoid this diff? If I keep my current namespacing, do I need to regenerate these files instead of manually editing them?

Reply:
If the namespace has changed, manually updating the designer should be fine. Re-genning of the designed file would result in the same output.